### PR TITLE
add prettier and eslint

### PR DESCRIPTION
**Pull request by vujevits**
### Problem
Frontend code was using mixed style, also eslint was missing.

### Solution
Added prettier and eslint to @tender/app`.

### Todo later
- integrate with CI
- precommit hook?
- [x] make sure it works with VSCode too

### CODE REVIEW

File: packages/app/src/App.tsx [hunk 1]
```diff
@@ -1,48 +1,52 @@
...
 };
 
-function App() {
+const App: FC = () => {
+  const b = 4;
+
+  useEffect(() => {
+    console.log(b);
+  }, []);
+
```
Comment (kyriediculous) [inline packages/app/src/App.tsx:32]: Can remove this I think ?

**Review by kyriediculous — CHANGES_REQUESTED** [review]
Looks good to me, just some leftover code that stuck around that needs to be removed I think. 

Since you're now on the github team you could clone this repository itself so your next PRs will be built by Vercel as a CI check.

File: packages/app/src/App.tsx (hunk 1 above)
Comment (vujevits) [reply packages/app/src/App.tsx:32]: good that you caught that, actually the eslint rule is still not set, I added this for testing 👍

**Comment by kyriediculous** [comment]
Looks good ! Let's rebase on top of master and resolve the merge conflict

**Pull request merged**